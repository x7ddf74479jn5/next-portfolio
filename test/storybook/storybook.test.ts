import initStoryshots from "@storybook/addon-storyshots";
import { setupWindow } from "test/test-utils";

setupWindow();
initStoryshots();
