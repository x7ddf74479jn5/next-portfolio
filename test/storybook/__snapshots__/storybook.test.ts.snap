// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots about/SkillItem Default 1`] = `
<div
  className="wrapper"
>
  <h3
    className="head"
  >
    ヘッダー
  </h3>
  <div
    className="data"
  >
    テキスト欄
  </div>
</div>
`;

exports[`Storyshots about/TimelineList Default 1`] = `
<ul
  className="timeline"
>
  <li>
    <time
      dateTime="Time 1"
    >
      Time 1
    </time>
    <div
      className="timelineIcon"
    />
    <div
      className="timelineBox"
    >
      <h3>
         
        Title 1
         
      </h3>
      <p>
         
         Description 1
         
      </p>
    </div>
  </li>
  <li>
    <time
      dateTime="Time 2"
    >
      Time 2
    </time>
    <div
      className="timelineIcon"
    />
    <div
      className="timelineBox"
    >
      <h3>
         
        Title 2
         
      </h3>
      <p>
         
         Description 2
         
      </p>
    </div>
  </li>
  <li>
    <time
      dateTime="Time 3"
    >
      Time 3
    </time>
    <div
      className="timelineIcon"
    />
    <div
      className="timelineBox"
    >
      <h3>
         
        Title 3
         
      </h3>
      <p>
         
         Description 3
         
      </p>
    </div>
  </li>
</ul>
`;

exports[`Storyshots chatbot/Answer Default 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-1"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    Content
  </span>
</button>
`;

exports[`Storyshots chatbot/AnswersList Default 1`] = `
<div
  className="gridAnswer"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Content 1
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Content 2
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Content 3
    </span>
  </button>
</div>
`;

exports[`Storyshots chatbot/Chat Answer 1`] = `
<li
  className="MuiListItem-root chatReverse MuiListItem-gutters"
  disabled={false}
>
  <div
    className="MuiListItemAvatar-root"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circular"
    >
      <img
        alt="icon"
        className="MuiAvatar-img"
        src="/img/icons/no-profile.png"
      />
    </div>
  </div>
  <div
    className="chatBubble"
  >
    This text is an answer
  </div>
</li>
`;

exports[`Storyshots chatbot/Chat Question 1`] = `
<li
  className="MuiListItem-root chatRow MuiListItem-gutters"
  disabled={false}
>
  <div
    className="MuiListItemAvatar-root"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circular"
    >
      <img
        alt="icon"
        className="MuiAvatar-img"
        src="/img/icons/pandashark_180-180.png"
      />
    </div>
  </div>
  <div
    className="chatBubble"
  >
    This text is a question
  </div>
</li>
`;

exports[`Storyshots chatbot/Chatbot Chatbot 1`] = `
<section
  className="section"
>
  <div
    className="box"
  >
    <ul
      className="MuiList-root makeStyles-chats-3 MuiList-padding"
      id="scroll-area"
    />
    <div
      className="gridAnswer"
    />
  </div>
</section>
`;

exports[`Storyshots chatbot/Chats Default 1`] = `
<ul
  className="MuiList-root makeStyles-chats-4 MuiList-padding"
  id="scroll-area"
>
  <li
    className="MuiListItem-root chatRow MuiListItem-gutters"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular"
      >
        <img
          alt="icon"
          className="MuiAvatar-img"
          src="/img/icons/pandashark_180-180.png"
        />
      </div>
    </div>
    <div
      className="chatBubble"
    >
      This text is the first question
    </div>
  </li>
  <li
    className="MuiListItem-root chatReverse MuiListItem-gutters"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular"
      >
        <img
          alt="icon"
          className="MuiAvatar-img"
          src="/img/icons/no-profile.png"
        />
      </div>
    </div>
    <div
      className="chatBubble"
    >
      This text is the first answer
    </div>
  </li>
  <li
    className="MuiListItem-root chatRow MuiListItem-gutters"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular"
      >
        <img
          alt="icon"
          className="MuiAvatar-img"
          src="/img/icons/pandashark_180-180.png"
        />
      </div>
    </div>
    <div
      className="chatBubble"
    >
      This text is the second question
    </div>
  </li>
  <li
    className="MuiListItem-root chatReverse MuiListItem-gutters"
    disabled={false}
  >
    <div
      className="MuiListItemAvatar-root"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular"
      >
        <img
          alt="icon"
          className="MuiAvatar-img"
          src="/img/icons/no-profile.png"
        />
      </div>
    </div>
    <div
      className="chatBubble"
    >
      This text is the second answer
    </div>
  </li>
</ul>
`;

exports[`Storyshots common/BreadCrumbs Portfolio 1`] = `
<ul
  aria-label="Breadcrumbs"
  className="textBreadcrumbs"
>
  <li>
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      HOME
    </a>
     
    <span>
      ＞
    </span>
  </li>
  <li>
    <a
      href="/samples"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      SAMPLES
    </a>
     
    <span>
      ＞
    </span>
  </li>
  <li>
    <span
      aria-current="page"
    >
      PORTFOLIO
    </span>
     
  </li>
</ul>
`;

exports[`Storyshots common/NextArrowButton Default 1`] = `
<a
  className="btnRound iconArrowNext"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  role="button"
>
  Button
</a>
`;

exports[`Storyshots common/NextArrowButton On Dark 1`] = `
<a
  className="btnRoundOnDark iconArrowNext"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  role="button"
>
  Button
</a>
`;

exports[`Storyshots common/PandasharkLogo Default 1`] = `
<a
  aria-label="パンダシャーク"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  target="_self"
>
  <img
    alt="Pandashark's icon"
    height={64}
    src="/img/icons/pandashark_logo_rectangle.webp"
    width={128}
  />
</a>
`;

exports[`Storyshots common/PrimaryButton Default 1`] = `
<button
  className="btnRound"
  disabled={false}
>
  Button
</button>
`;

exports[`Storyshots common/TitleArea Portfolio 1`] = `
<section
  className="section"
>
  <div
    className="sectionTitleArea fadeInLeft"
  >
    <div
      className="headlineSub fadeInUp"
    >
      <h2>
        Portfolio
      </h2>
      <p>
        ポートフォリオ
      </p>
      <ul
        aria-label="Breadcrumbs"
        className="textBreadcrumbs"
      >
        <li>
          <a
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            HOME
          </a>
           
          <span>
            ＞
          </span>
        </li>
        <li>
          <a
            href="/samples"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            SAMPLES
          </a>
           
          <span>
            ＞
          </span>
        </li>
        <li>
          <span
            aria-current="page"
          >
            PORTFOLIO
          </span>
           
        </li>
      </ul>
    </div>
  </div>
</section>
`;

exports[`Storyshots contact/ContactFormContainer Default 1`] = `
<form
  className="p-forms"
  name="contactForm"
  onSubmit={[Function]}
>
  <div
    className="formsInput"
  >
    <p>
      <label
        htmlFor="name"
      >
        お名前 *
      </label>
      <input
        id="name"
        name="name"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="熊猫 鮫"
        type="text"
      />
    </p>
  </div>
  <div
    className="formsInput"
  >
    <p>
      <label
        htmlFor="email"
      >
        メールアドレス *
      </label>
      <input
        id="email"
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="your@example.com"
        type="email"
      />
    </p>
  </div>
  <div
    className="formsInput"
  >
    <p>
      <label
        htmlFor="category"
      >
        お問い合わせ種別
      </label>
      <select
        id="category"
        name="category"
        onBlur={[Function]}
        onChange={[Function]}
      >
        <option
          value=""
        >
          
        </option>
        <option
          value="Webサイト制作について"
        >
          Webサイト制作について
        </option>
        <option
          value="Webアプリ開発について"
        >
          Webアプリ開発について
        </option>
        <option
          value="Webサイト模写について"
        >
          Webサイト模写について
        </option>
        <option
          value="その他"
        >
          その他
        </option>
      </select>
    </p>
  </div>
  <div
    className="formsInput"
  >
    <p>
      <label
        htmlFor="description"
      >
        お問い合わせ内容 *
      </label>
      <textarea
        id="description"
        name="description"
        onBlur={[Function]}
        onChange={[Function]}
        rows={10}
      />
    </p>
  </div>
  <button
    className="btnRound"
    disabled={false}
    id="confirm-btn"
    type="submit"
  >
    内容を確認する
  </button>
</form>
`;

exports[`Storyshots contact/FormButton Apply 1`] = `
<button
  className="btnApply"
  disabled={false}
>
  送信する
</button>
`;

exports[`Storyshots contact/FormButton Cancel 1`] = `
<button
  className="btnCancel"
  disabled={false}
>
  修正する
</button>
`;

exports[`Storyshots contact/ModalPopup Default 1`] = `
<div
  className="c-modal-popup"
  id="popup"
>
  <div
    className="modalPopupBody"
  >
    <h2
      className="modalPopupTitle"
      id="modal-title"
    >
      入力内容の確認
    </h2>
    <div
      className="modalPopupMain"
      id="popup-main"
    >
      <div
        id="mail-content"
      >
        <p>
          お名前：
        </p>
        <p>
          Name
        </p>
        <p>
          メールアドレス：
        </p>
        <p>
          test@email.com
        </p>
        <p>
          お問い合わせ種別：
        </p>
        <p>
          その他
        </p>
        <p>
          お問い合わせ内容：
        </p>
        <p>
          これはテストです。
        </p>
      </div>
    </div>
    <div
      className="gridRow"
    >
      <button
        className="btnCancel"
        disabled={false}
        id="cancel-btn"
        onClick={[Function]}
        type="button"
      >
        修正する
      </button>
      <button
        className="btnApply"
        disabled={false}
        id="apply-btn"
        onClick={[Function]}
        type="button"
      >
        送信する
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots footer/Accordion Default 1`] = `null`;

exports[`Storyshots footer/Foot Default 1`] = `
<section
  className="foot"
>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Item 1
      </a>
    </li>
    <li>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Item 2
      </a>
    </li>
    <li>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Item 3
      </a>
    </li>
  </ul>
</section>
`;

exports[`Storyshots footer/Footer Default 1`] = `
<footer
  className="footer"
>
  <div
    className="main"
  >
    <div
      className="mainInner"
    >
      <div
        className="wrapper"
      >
        <section
          className="logoArea"
        >
          <div
            className="productLogo"
          >
            <a
              aria-label="パンダシャーク"
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              target="_self"
            >
              <img
                alt="Pandashark's icon"
                height={64}
                src="/img/icons/pandashark_logo_rectangle.webp"
                width={128}
              />
            </a>
          </div>
        </section>
        <section
          className="sitemap"
        >
          <ul
            className="footerLinks"
          >
            <li />
            <li />
            <li />
          </ul>
        </section>
      </div>
      <section
        className="foot"
      >
        <ul>
          <li>
            <a
              href="/terms"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              利用規約
            </a>
          </li>
          <li>
            <a
              href="/privacy"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              プライバシーポリシー
            </a>
          </li>
        </ul>
      </section>
    </div>
  </div>
</footer>
`;

exports[`Storyshots header/Header Header 1`] = `
<header
  className="header"
>
  <div
    className="headerInner"
  >
    <h1
      className="productLogo"
    >
      <a
        aria-label="パンダシャーク"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        target="_self"
      >
        <img
          alt="Pandashark's icon"
          height={64}
          src="/img/icons/pandashark_logo_rectangle.webp"
          width={128}
        />
      </a>
    </h1>
    <button
      aria-label="ナビゲーション メニューの切り替え"
      className="navToggleButton"
      data-about-nav-toggle={true}
      id="nav-toggle-button"
      onClick={[Function]}
    >
      <svg
        aria-hidden={true}
        aria-label="ナビゲーション メニューを開く\\""
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </button>
    <nav
      className="headerNavTop"
    >
      <h1
        className="productLogo"
      >
        <a
          aria-label="パンダシャーク"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          target="_self"
        >
          <img
            alt="Pandashark's icon"
            height={40}
            src="/img/icons/pandashark_logo_rectangle.webp"
            width={80}
          />
        </a>
      </h1>
      <ul>
        <li>
          <a
            className="linkItem"
            href="/about"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            About
          </a>
        </li>
        <li>
          <a
            className="linkItem"
            href="/samples"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Samples
          </a>
        </li>
        <li>
          <a
            className="linkItem"
            href="/contact"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Contact
          </a>
        </li>
        <li>
          <a
            className="linkItem"
            href="https://twitter.com/pandashark6"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="twitter"
              src="/img/icons/twitter-logo.svg"
            />
            <span>
              Twitter
            </span>
          </a>
        </li>
        <li>
          <a
            className="linkItem"
            href="https://github.com/x7ddf74479jn5"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            <img
              alt="github"
              src="/img/icons/github-logo.svg"
            />
            <span>
              GitHub
            </span>
          </a>
        </li>
        <li>
          <button
            className="buttonItem"
            onClick={[Function]}
          >
            <img
              alt="chatbot"
              src="/img/icons/question_answer_black.svg"
            />
            <span>
              Chatbot
            </span>
          </button>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;

exports[`Storyshots header/HeaderButtonItem Chatbot 1`] = `
<button
  className="buttonItem"
>
  <img
    alt="chatbot"
    src="/img/icons/question_answer_black.svg"
  />
  <span>
    Chat Bot
  </span>
</button>
`;

exports[`Storyshots header/HeaderLinkItem Default 1`] = `
<a
  className="linkItem"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  Item
</a>
`;

exports[`Storyshots header/HeaderLinkItem Git Hub 1`] = `
<a
  className="linkItem"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  <img
    alt="github"
    src="/img/icons/github-logo.svg"
  />
  <span>
    GitHub
  </span>
</a>
`;

exports[`Storyshots header/HeaderLinkItem Twitter 1`] = `
<a
  className="linkItem"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  <img
    alt="twitter"
    src="/img/icons/twitter-logo.svg"
  />
  <span>
    Twitter
  </span>
</a>
`;

exports[`Storyshots header/NavToggleButton Close 1`] = `
<button
  aria-label="ナビゲーション メニューの切り替え"
  className="navToggleButton"
  data-about-nav-toggle={true}
  id="nav-toggle-button"
>
  <svg
    aria-hidden={true}
    aria-label="ナビゲーション メニューを開く\\""
    className="MuiSvgIcon-root"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
    />
  </svg>
</button>
`;

exports[`Storyshots header/NavToggleButton Open 1`] = `
<button
  aria-label="ナビゲーション メニューの切り替え"
  className="navToggleButtonOpen"
  data-about-nav-toggle={true}
  id="nav-toggle-button"
>
  <svg
    aria-hidden={true}
    aria-label="ナビゲーション メニューを閉じる"
    className="MuiSvgIcon-root"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
    />
  </svg>
</button>
`;

exports[`Storyshots layouts/Spacer LG 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="lg"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots layouts/Spacer MD 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="md"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots layouts/Spacer SM 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="sm"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots layouts/Spacer XL 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="xl"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots layouts/Spacer XS 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="xs"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots layouts/Spacer XXS 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
  <div
    className="xxs"
  />
  <div
    style={
      Object {
        "backgroundColor": "#41b6e6",
        "height": "100px",
      }
    }
  />
</div>
`;

exports[`Storyshots modal/Backdrop Opacity 10 1`] = `
<div
  className="backdrop"
/>
`;

exports[`Storyshots modal/Backdrop Opacity 70 1`] = `
<div
  className="backdrop_70"
/>
`;

exports[`Storyshots modal/ModalCloseButton Default 1`] = `
<button
  className="closeButton"
>
  ×
</button>
`;

exports[`Storyshots projects/ServiceItem Default 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <article
    className="gridListItemFloated"
  >
    <h3>
      WEBサイト制作
    </h3>
    <small>
      Website Production
    </small>
    <img
      alt="Website Production"
      className="mediaThumb"
      height={288}
      src="/img/services/website-production.jpeg"
      width={512}
    />
    <p>
      コーポレートサイト、個人ポートフォリオ、ランディングページなどWebサイト制作全般を承ります。検索流入やSNS流入を強化する「SEOを意識したサイト」を作成できます。
    </p>
  </article>
</div>
`;

exports[`Storyshots samples/ExternalLink Default 1`] = `
<a
  href=""
  rel="noreferrer"
  target="_blank"
>
  <span
    className="externalLink"
  >
    Label
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
      />
    </svg>
  </span>
</a>
`;

exports[`Storyshots samples/SampleCard Portfolio 1`] = `
<div
  style={
    Object {
      "maxWidth": "500px",
    }
  }
>
  <div
    className="sampleCard"
  >
    <a
      className="testLineNone"
      href="/samples/portfolio"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="mediaThumb"
      >
        <img
          alt="eye catch of portfolio"
          layout="fill"
          src="/img/samples/portfolio-eyecatch.webp"
        />
      </div>
      <div
        className="listSamplesDetail"
      >
        <h3
          className="title"
        >
          ポートフォリオ
        </h3>
        <p
          className="category"
        >
          Web制作｜デザイン｜レスポンシブ｜アニメーション
        </p>
        <div
          className="icon"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 5H11l5 7-5 7h4.5l5-7z"
            />
            <path
              d="M8.5 5H4l5 7-5 7h4.5l5-7z"
            />
          </svg>
        </div>
      </div>
    </a>
  </div>
</div>
`;

exports[`Storyshots utils/ColorChart Color Chart 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexWrap": "wrap",
      "gap": "10px",
    }
  }
>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      light
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #f0f0f0
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#f0f0f0",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      dark
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #001871
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#001871",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      mainDark
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #444
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#444",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      main
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #4c4c4c
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#4c4c4c",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      mainLight
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #777
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#777",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      primary
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #41b6e6
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#41b6e6",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      primaryLight
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #a2dbf3
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#a2dbf3",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      secondary
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #ffb549
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#ffb549",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      secondaryLight
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #ffca7c
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#ffca7c",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
  <div>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      accent
    </p>
    <p
      style={
        Object {
          "margin": "0",
        }
      }
    >
      #ff585d
    </p>
    <div
      style={
        Object {
          "backgroundColor": "#ff585d",
          "height": "120px",
          "width": "120px",
        }
      }
    />
  </div>
</div>
`;
